pkg/csigrpc/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Instead of putting this library under `pkg/csigrpc/...` please move this to the top under `protosanitizer/...` (similar to packages in https://github.com/kubernetes/utils).

Since this is a util only repo, it does not need `cmd`, `pkg`, etc directories. It will contain 1 or more go packages that other consumers will import and build in to binaries to ship.

Reply:
Having it at the top make sense.

I wanted to grow the content of the package over time and also include other gRPC-related functions (hence the current package name), but I can't really come up with an argument why that has to be in the same package.

So `protosanitizer` it is.